### DISCUSSION TITLE
Qual o seu método de estudo?

### DISCUSSION THREAD

**Discussion by eduardojmatos** [frontendbr/forum#273] [Discussão]
Olá gente! Queria abrir essa discussão pra entender melhor o que cada um prefere.

Eu, por exemplo, normalmente leio blog posts e alguns livros. Vídeos tenho tentado, mas na correria do dia-a-dia acaba ficando ruim acompanhar alguns canais.

Aula presencial faz muuuito tempo que não faço, e também virtual. 

E aí? O que curtem na hora de estudar algo novo?

**Comment by diessica** [comment]
Você não está se referindo a quais livros, sites ou canais, e sim ao **método**, certo? Eu tenho uma opinião bem definida em relação a isso.

Para mim, a leitura é o principal meio para adquirir qualquer tipo de conhecimento, e o que sustenta essa afirmação é o **tempo**. Eu aprendo mais **por minuto** quando estudo lendo, seja documentação ou livros técnicos, e isso pois a escrita geralmente é uma forma de comunicação mais filtrada, o que  faz com que você adquira informação de mais relevância com mais clareza. Há uma seleção melhor de palavras e tópicos em um período menor de tempo.

Vídeos são interessantes para a prática ou informações mais pontuais e específicas de um autor em particular (exemplo: Dan Abramov ensinando Redux, membro do _core team_ do Ember ensinando Ember...), mas, para a teoria genérica, sinceramente.... Consumir informação através da  leitura é mais rápido. (O problema é que a fadiga também chega mais rápido, o que não acontece com vídeos. Nesse caso,  é importante conciliar os métodos de modo que você aproveite melhor o tempo.)

Cursos  sempre fazem um papel melhor **orientando** do que propriamente ensinando, se você notar. Eu diria que, basicamente, o método escolhido depende muito da sua capacidade de organização e orientação e do quão rápido você cansa (sim, estudar cansa muito!).

**Comment by eduardojmatos** [comment]
@diessica isso, ao método mesmo =D

Eu realmente prefiro leitura ao invés de vídeos, mas é o que vc falou. Uma hora leitura cansa bastante, e o vídeo fica perfeito pra esses momentos.

**Comment by felipemrodrigues** [comment]
Eu gosto de ler, mas acredito que vídeos sejam mais eficazes.

Vídeos não conseguem prender minha atenção.

Nada é melhor do que fazer mesmo.

**Comment by juao** [comment]
Leitura e projetos (mão na massa), praticando.

Ultimamente tenho tentado com vídeos, mas não rola uma sinergia bacana, não sei se é pelo motivo de não ser ao vivo.

**Comment by gusribeiro** [comment]
Antigamente eu lia muitos sites e blogs pois livros nunca me prenderam a atenção necessária para o aprendizado.
O mais eficiente mesmo, para mim, é o 'mão na massa'. Só assim eu consigo absorver o conhecimento e saber o que eu realmente estou fazendo.
Hoje, quando lembro, tento assistir as video aulas no youtube, mas também está difícil prender atenção :)

**Comment by felipeuntill** [comment]
Gosto muito pouco do conteúdo compartilhado no Brasil por duas razões:

1- Quem publica conteúdo no Brasil, na grande maioria das vezes ou quer promover a si mesmo ou sua empresa portanto, a passagem de conteúdo normalmente é introdutória e coisas importantes não são explicadas para que o respectivo lecionador guarde aquele diferencial e continue a frente do mercado(Note que não generalizei, existem algumas poucas pessoas de bom coração).

2- A aderência a novas tecnologias no Brasil é um pouco retarda, normalmente nós utilizamos aqui oque já virou tendência lá fora, portanto, focar na onda do momento daqui implica em estar a pelo menos duas marolas atrás do que acontece lá fora. 
Claro que esta distância caiu muito nos últimos anos mas, no Brasil culturalmente nós somente utilizamos soluções extremamente robustas e antiquadas em projetos de grande porte e a experiência dos profissionais nas novas tecnologias vai se resumir a projetos de pequeno porte.

Manter-se atualizado no cenário atual onde temos um framework novo por semana está quase impossível e como sou full-stack e trabalho com várias linguagens diariamente(C#, Ruby, Python, Swift, Javascript) tenho que acompanhar os mais distintos portais e comunidades. 

O fato é tem sido impossível colocar em pratica tudo que vejo, estando com as atenções voltadas para NodeJS e Swift. Para as demais linguagens eu apenas acompanho noticias e posts dos indivíduos que considero relevantes.

Parei de trabalhar de carro para aumentar meu tempo produtivo e utilizo transporte publico onde assisto os cursos do **Pluralsight**, **Udemy** e **Udacity** quais acho muito bons e relevantes, também leio muitos artigos pois considero interessante aprender com os erros e problemas alheios antes de cometer os meus. 

Nas redes sociais só sigo minha noiva e meus 3 melhores amigos para não perder tempo, todas as demais postagens são sobre empreendedorismo ou TI e quando chego em casa tenho a meta de programar no mínimo 3 horas por dia em projetos pessoais, não importando o horário que eu vá dormir.

**Comment by danilovaz** [comment]
Concordo com a @diessica em relação à aprender com base na leitura. Sejam livros ou posts, o conteúdo que se pode passar na leitura _pode ser_ mais rico em detalhes.

Eu também me encaixo no grupo que lê documentações, posts (blogs e medium's) e discussões (aqui, twittere discuss). Na maior parte das vezes, são suficientes para sanar uma dúvida ou aprender algo novo. Mas em algumas situações sinto falta de uma conversa cara a cara, na hora de aprender algo novo. 

O que no caso um curso online, talvez, proporcione mais relevância ao ensino. Por exemplo, estou aprendendo Ember.js esse ano, leio a documentação diariamente, participo do [discuss deles](http://discuss.emberjs.com/) e leio blogs sobre o framework. 
Porém, na cidade que eu moro, não tem ninguém pra eu trocar ideia sobre o hamster. Para isso, cursos onlines são uma mão na roda, porque você pode ter uma explicação resumida em vídeo e se tiver a opção de hangouts de dúvidas, melhor ainda.

No meu caso a leitura é o primeiro e melhor método para aprender. Mas também assisto vídeos e até gosto de debater com outras pessoas o que aprendi, quando ouço alguém explicando algo, automaticamente acabo buscando um pensamento crítico para a afirmação do indivíduo. Senão vira um aprendizado no estilo "copy/paste" para o cérebro em que "Algo é bom porque fulano disse que é". Por isso também tento assistir vídeos e fazer cursos online, mas como disseram: **tempo** é um negócio bem complicado rs.

Ah, acredito que Meetups se encaixam bem em uma forma de se aprender. Os debates gerados são proveitosos. Quem mora em São Paulo tem uma variedade incrível desses encontros, é uma boa oportunidade de tirar uma dúvida com alguém e colocar na mesa um pensamento que você tenha à respeito do que aprendeu. Moro um pouco longe de SP e como não tem Meetups desse nível aqui, o jeito é sair criando esses grupos e encontrando outros doidos pelo caminho pra ajudar :laughing:

**Comment by renatogalvones** [comment]
Na minha ótica, não adianta nenhuma forma de estudo se não houver a prática para "gravar" o conhecimento.

Nos videos e nos livros vc vê tudo funcionando. Na prática, vc vê dando erro e batendo a cabeça pra arrumar tendo que recorrer ao conteúdo pra sair de um problema e etc.

Então, minha sugestão é: seja qual for o método de estudo, a prática tem q existir.

**Comment by woliveiras** [comment]
Eu gosto de **leitura**.

Quando quero aprender algo novo vou lá e compro um livro sobre o assunto (não só relacionados a tecnologia, na prateleira tem desde HTML até Psicologia ou Matemática Discreta).

Quando não é algo tão importante ao ponto de ser necessário gastar um dim dim com livros eu procuro o máximo de conteúdo Online sobre o assunto, como Blogs, Wiki.

Uma coisa que foi importante pra definição da melhor maneira de estudar, pra mim, foi testar várias maneiras até achar a que eu mais curti.
Todos temos um ritmo diferente de estudos devido ao tempo disponível e até mesmo da nossa disciplina, disposição ou nível de preguiça (sim, todo mundo aqui tem preguiça, que eu sei :stuck_out_tongue_closed_eyes:).

Quando vejo um bom curso, como o do @fdaciuk sobre JS, eu até cogito se vira ou não fazer. Só que eu sou bem fraco com questão de ficar vendo vídeos, se não for algo dinâmico e que me desafie a tentar algo, eu não consigo manter o ritmo.

Algo que eu vi que não sou tão bom é manter a atenção por muito tempo em alguém falando em vídeos.
Se eu não posso perguntar na hora pra pessoa, eu não consigo manter a atenção necessária 😢 .

Hoje eu faço faculdade a distância, olha que maneiro pra quem não curte tanto vídeo aulas, e pra manter o ritmo e aprendizado eu vejo os vídeos e foco mais no conteúdo escrito que a faculdade disponibiliza + pesquisas próprias.

Gosto de técnicas para manter o foco e utilizo a Técnica de Pomodoro para estudar.

Fiz até um post sobre o assunto: https://woliveiras.com.br/posts/como-alcancar-objetivos-rapidamente-nos-estudos/

Eu mantinha uma prática de estudar uma linguagem por ano, para aprender novos conceitos, ler, ao menos, 3 livros técnicos no mesmo ano e criar pelo menos um _hello world_ com o que aprendi, porém hoje está meio complicado e tive de parar com isso pra focar em outras coisas, mas esse esquema era muito bom. 
Principalmente pelo ponto que o @renatogalvones falou: prática.

**Como praticar sem ter tempo?**

Quando eu não pratico algo, por falta de tempo (não poder priorizar aquilo) eu tento compartilhar o que aprendi pelo Blog ou entrando em comunidades do assunto e ajudando a galera que tem dúvidas.

Pra mim o que mais funciona, pra fixar o conteúdo, é praticar e compartilhar o que aprendeu. A dúvida de outras pessoas fortalece o que aprendemos na nossa mente.

**Comment by vitocaio** [comment]
Acredito que não existe nada melhor para aprender, se não um bom mentor rs.
Eu sempre busco leituras sobre os assuntos que desejo aprender ou vou atras das vídeo aulas, mas para mim, nada se compara com ter uma pessoa que manja muito do assunto e não tem preguiça de ensinar, acredito que quando conectamos nossas mentes com outra pessoa, absorvemos muito rápido as coisas.

**Comment by juao** [comment]
Vi a piramide de Glasser em uma palestra de VR dia desses:
![13903302_10154346518678490_138411892677002478_n](https://cloud.githubusercontent.com/assets/3719293/17446223/9cc8bdf8-5b1e-11e6-8c2d-c79d7f95621b.jpg)

**Comment by danilovaz** [comment]
@juao Não tinha conhecimento dessa pirâmide, show!

As três da base são algo que uma comunidade proporciona:

**95%:** Palestrar ou dar um workshop;
**80%:** Palestrar com demonstração ou praticar;
**70%:** Meetups e outros grupos de discussões

**Comment by felipefialho** [comment]
Essa piramide é do @willgm? :smile:

**Comment by clarabatt** [comment]
Eu sou muito dispersa então o que mais funciona pra mim é fazer **anotações**, dessa forma eu me obrigo a prestar mais atenção.

**Programação** - Procuro escrever as anotações em arquivos da própria linguagem, colocando as notas como comentário, assim eu já posso ir testando na hora.

**Livros** - Uso lápis pra grifar os trechos importantes e post-its para escrever alguma observação ou dúvida. 

**Artigos** - Salvo em PDF e assim posso grifar e adicionar notas, mas só faço isso com os que precisam de mais atenção. 

**Aulas** - Anoto no evernote ou no bloco de notas quando é por vídeo e na faculdade procuro escrever no caderno pra não perder a prática :laughing: 

**Vantagens** 
- Consigo me lembrar de partes específicas, que eu não me lembraria caso tivesse apenas assistido.
- Tenho uma mini documentação do assunto, não tendo que ficar procurando em qual parte do vídeo está ou o blog em que eu li determinada coisa, assim posso ir direto ao ponto.
- Poder ir testando exemplos na hora, no caso de programação.

**Desvantagens** 
- Posso perder um pouco mais de tempo parando o conteúdo para tomar nota, mas esse tempo é recuperado depois porque não tenho que ficar re-assistindo/lendo por falta de atenção ou porque  preciso de algo.

Não conhecia a pirâmide, acho que sem querer faço um método que é completão kkkkk

**Comment by Marcelosilva10** [comment]
Gostei dessa possibilidade da Piramide, alguém aqui usa o método promodoro?

**Comment by eduardojmatos** [comment]
@Marcelosilva10 eu tento usar, mas sempre esqueço de dar a pausa ou de sair dela hehehe

**Comment by felipeuntill** [comment]
@Marcelosilva10  Quando volto a utilizar viro uma maquina de produzir mas não sou regrado o sulficiente  pra manter por mais que dois dias, minha dica pra quem usa Mac é o [Pomodoro Timer](http://xwavesoft.com/pomodoro-timer-for-iphone-ipad-mac-os-x.html) que funciona que é uma beleza e ainda gera relatório de tudo.

**Comment by Marcelosilva10** [comment]
Eu tento que seguir algo nesse estilo de estudo, muita coisa para estudar acho que isso e um fator para não ter um foco ideal nos objetivos.
Inglês, Front end, a dificuldade e quantas horas pode ser regradas a isso sabe.

**Comment by bernardodiasc** [comment]
Gosto muito dessa pirâmide de Glasser.

Minha metodologia é de ler muito, salvo os links numa lista-diário pra consulta futura, quando sinto necessidade parto pra experimentação, qualquer POC serve. A ideia de experimentar é botar à prova o aprendizado e tentar quebrar as coisas, validar as ideias e conceitos digamos assim, além de fixar o aprendizado do que é útil. Trocar ideias com outros também ajuda bastante a ver aqueles detalhes que só a experiência encontra e diferentes pontos de vista.

De fato, onde o aprendizado é mais eficaz é quando lecionando. Elaborar material didático é um belo desafio, ensinar é incrivelmente eficiente pois você precisa tanto dominar a técnica quanto dominar a didática do assunto. Ensinar turmas ou alunos individualmente é ainda mais poderoso que palestrar.

**Comment by notnotgabriel** [comment]
Gostei muito dessa pirâmide de Glasser :smile: 
Em geral eu me dou melhor com vídeos, os do CodeSchool são excelentes! (momento jabá).

Tem um vídeo no youtube com dicas bastante interessantes sobre as técnicas que mais dão certo:
https://www.youtube.com/watch?v=sXmKaZ-9fWc

**Comment by dionarodrigues** [comment]
Meu método é bem parecido com o da @clarabatt : preciso anotar tudo pra absorver o conhecimento. E, claro, ir testando aquilo que venho aprendendo.
